tests/ArgentPonyWarcraftClient.Tests/GameDataApi/AchievementApiTests.cs

```diff
@@ -59,6 +59,8 @@ public async Task GetAchievementMediaAsync_Gets_AchievementMedia()
 
             RequestResult<AchievementMedia> result = await warcraftClient.GetAchievementMediaAsync(6, "static-us");
             Assert.NotNull(result.Value);
+
+            result.ShouldMatchJsonContent(Resources.AchievementMediaResponse);
```

Comment:
Trial balloon?  I'm assuming your intention is to roll this approach out to pretty much all of the integration tests over time.

Reply:
Yeah, I'll probably undo this change. My mindset was "well I fixed the `Asset` class so I'll add this check to the Achievement tests" since I knew it was there. But it would better to just do all of the tests with `Asset` instead of having a rogue additional change like this.